ironfish/src/primitives/block.ts

```diff
@@ -47,13 +47,15 @@ export class Block {
       t.takeReference()
     }
 
-    try {
-      return callback()
-    } finally {
+    const result = callback()
```

Comment:
Why not just await callback?

Reply:
This one is easier to change, but see other comment for reasoning